superset-frontend/src/dashboard/components/nativeFilters/FiltersConfigModal/FiltersConfigForm/getControlItemsMap.tsx

```diff
@@ -114,7 +118,7 @@ export default function getControlItemsMap({
             initialValue={initColumn}
             label={
               <StyledLabel>
-                {mainControlItem.config?.label || t('Column')}
+                {mainControlItem.config?.label || t('Column label')}
```

Comment:
```suggestion
                {mainControlItem.config?.label || t('Column for labels')}
```

With this change, i would like to say "column used for label filter items"